src/sales/order-workflow.md

```diff
@@ -8,13 +8,13 @@ Step 1: Place Order
 : The checkout process begins when the shopper clicks the Go to Checkout button on the shopping cart page or [reorders]({% link sales/reorders-allow.md %}) directly from their customer account.
 
 Step 2: Order Pending
-: In the Orders grid, the status of the sales order is initially `Pending`. Payment has not been processed, and the order can still be edited or canceled.
+: The status of the sales order is initially `Pending`. Payment has not been processed, and the order can still be edited or canceled. This will occur when payment processor module is configured for authorization mode.
 
 Step 3: Receive Payment
-: The status of the order changes to `Processing` when payment is received or authorized. Depending on the payment method, you might receive notification when the transaction is authorized or processed.
+: The status of the order changes to `Processing` when payment is received or authorized. Depending on the payment method, you might receive notification when the transaction is authorized or processed. This will occur automatically when payment processor module is configured for capture or intent sale mode.
 
 Step 4: Invoice Order
-: An order is typically invoiced after payment is received. Some payment methods generate an invoice automatically when payment is authorized and captured. The payment method determines which invoicing options are needed for the order. After the invoice is generated and submitted, a copy is sent to the customer.
+: An order is typically invoiced after payment is received. Some payment methods generate an invoice automatically when payment is authorized and captured. The payment method determines which invoicing options are needed for the order. After the invoice is generated and submitted, a copy is sent to the customer. This will occur automatically when payment processor module is configured for capture or intent sale mode.
```

Comment:
Last sentence:  What does "this" refer to -- the invoicing of the order?  or automatic invoicing?   Same comment regarding "payment processor module" = payment method

Reply:
It seems to me you can combine the last two sentences for clarity....

Something like: "After the invoice is generated and submitted, a copy is automatically sent to the customer If the payment method is configured for capture or intent sale mode." or "If the payment method is configured for capture or intent sale mode, a copy of the invoice is automatically sent to the customer."

Also, if "capture" or "intent sale mode" are actually payment method statuses or configurations I have typically formatted those as code, as you do below with " `Complete` ". Unsure if that is needed here, but wanted to call it out. Is intent sale a mode? I guess I'm confused on this verbiage.